scripts/newmoose.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+function printusage {
+    echo "Usage:    newapp.sh <type> <AppName>"
+    echo ""
+    echo "    <type> must be either "app" or "module"."
+    echo "    <AppName> should be given in CapCase format."
+}
+
+if [[ "$1" == "-h" || $# != 2 ]]; then
+    printusage
+    exit 1
+fi
+
+if [[ "$1" != "app" && "$1" != "module" ]]; then
+    echo "error: invalid type given" >&2
+    printusage
+    exit 1
+fi
+
+# set old/new app name variables
+srcname='Stork'
+dstname=$2
+kind=$1
+srcnamelow=$(echo "$srcname" | awk '{print tolower($0)}')
+srcnameup=$(echo "$srcname" | awk '{print toupper($0)}')
+dstnamelow=$(echo "$dstname" | awk '{print tolower($0)}')
+dstnameup=$(echo "$dstname" | awk '{print toupper($0)}')
+dir="$dstnamelow"
+if [[ -z $MOOSE_DIR ]]; then
+    MOOSE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+fi
```

Comment:
There needs to be more checking for `MOOSE_DIR` here... like more verification that it actually points at MOOSE.  I'm worried that the script might do some crazy junk if `MOOSE_DIR` were wrong!